auth_service/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM --platform=linux/amd64 openjdk:17-jdk-alpine
```

Hadolint (2.12.0)

[warning] Do not use --platform flag with FROM

(DL3029)